packages/web-react/src/types/shared/style.ts

```diff
@@ -1,6 +1,18 @@
 import { CSSProperties } from 'react';
+import { SpacingStyleProp } from '../../constants';
+import { BreakpointToken, SpaceToken } from './tokens';
 
-export interface StyleProps {
+export const STYLE_SPACING_AUTO = 'auto';
+export type StyleSpacingAuto = 'auto';
```

Comment:
```suggestion
export type StyleSpacingAuto = typeof STYLE_SPACING_AUTO;
```

Reply:
export type StyleSpacingAuto = 'auto' as const

and then the typeof works. Thanks!